sensing/gnss_poser/README.md

```diff
@@ -10,10 +10,10 @@ The `gnss_poser` is a node that subscribes gnss sensing messages and calculates
 
 ### Input
 
-| Name             | Type                          | Description                                                                                                     |
-| ---------------- | ----------------------------- | --------------------------------------------------------------------------------------------------------------- |
-| `~/input/fix`    | `sensor_msgs::msg::NavSatFix` | gnss status message                                                                                             |
-| `~/input/navpvt` | `ublox_msgs::msg::NavPVT`     | position, velocity and time solution. [click here for more details](https://github.com/KumarRobotics/ublox.git) |
+| Name                           | Type                                                       | Description                                                                                                                    |
+| ------------------------------ | ---------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------ |
+| `~/input/fix`                  | `sensor_msgs::msg::NavSatFix`                              | gnss status message                                                                                                            |
+| `~/input/autoware_orientation` | `autoware_sensing_msgs::msg::gnss_ins_orientation_stamped` | orientation [click here for more details](https://github.com/autowarefoundation/autoware_msgs/tree/main/autoware_sensing_msgs) |
```

Comment:
```suggestion
| `~/input/autoware_orientation` | `autoware_sensing_msgs::msg::GnssInsOrientationStamped` | orientation [click here for more details](https://github.com/autowarefoundation/autoware_msgs/tree/main/autoware_sensing_msgs) |
```

Reply:
Thanks for your answer. @YamatoAndo . I updated

Reply:
Fixed! https://github.com/autowarefoundation/autoware.universe/pull/1419/commits/9d54085e4172465a1965c3af4b13e8c3871930ce